<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineSaveDialog</class>
 <widget class="QDialog" name="EngineSaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warning</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="2">
    <widget class="QPushButton" name="mCancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="mSaveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="mWarningLabel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>You have made changes to this Engine.  Please one of the following options:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="mIgnoreChangesButton">
     <property name="text">
      <string>Ignore Changes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
